packages/broker/src/StreamMetrics.ts

```diff
@@ -124,8 +125,12 @@ export class StreamMetrics {
                     this.report.network.bytesToPeersPerSec = (metricsReport.metrics.WebRtcEndpoint.outSpeed as any).rate || 0
                     this.report.network.bytesFromPeersPerSec = (metricsReport.metrics.WebRtcEndpoint.inSpeed as any).rate || 0
                     this.report.network.connections = metricsReport.metrics.WebRtcEndpoint.connections || 0
-                    this.report.storage.bytesWrittenPerSec = (metricsReport.metrics['broker/cassandra']) ? metricsReport.metrics['broker/cassandra'].writeBytes : 0
-                    this.report.storage.bytesReadPerSec = (metricsReport.metrics['broker/cassandra']) ? metricsReport.metrics['broker/cassandra'].readBytes : 0
+
+                    const cassandraWrittenBytesPerMs = metricsReport.metrics['broker/cassandra'].writtenBytes as {rate: number,total: number, last: number}
+                    this.report.storage.bytesWrittenPerSec = cassandraWrittenBytesPerMs.rate / 1000 || 0
```

Comment:
Are we sure that the `rate` provided by the `MetricsContext` is in milliseconds? I recall it being in seconds with a window size of 5.

Reply:
After checking the measurements are provided in seconds, with the logic around the 5s window being handled before the metrics are populated